CMSgov/easi-app
# NOREF/SQL Scripts package

## Changes and Description

This is to streamline embedding SQL files for future use. As we go forward and touch SQL, new queries can be refactored to live in this package. Exporting these queries as a separate package gives us greater flexibility, and allows us to more granularly control where queries are accessed.

- Create a separated package for SQL Scripts
   - `sql_scripts.go` to hold package comments
   - Exported variables accessible to allow accessing scripts by logical types.
      - TRBRequest
          - Create
          - Update
          - GetByID
          - CollectionGet
          - CollectionGetByUserAndArchivedState
      - TRBRequestForm
          - Create    
- Existing Store references to these queries in storage now reference the exported variable



## How to test this change
1. Confirm that Unit tests pass
2. Make a TRB request form and confirm it works as expected


## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
